templates/docs/document.html

```diff
@@ -30,11 +30,11 @@ <h1>{{ document.title }}</h1>
       {{ document.body_html | safe }}
       <hr />
       <p><i>Last updated {{ document.updated }}.</i></p>
-      <div class="p-notification--information">
+      <!-- <div class="p-notification--information">
```

Comment:
They don't want it to be collaborative yet, but in the future we will add this.

Reply:
Can you remove the notification until ready?